src/relay/transforms/device_annotation.cc

```diff
@@ -574,7 +574,7 @@ Pass RewriteAnnotatedOps(int fallback_device) {
     return Downcast<Function>(relay::RewriteAnnotatedOps(f, fallback_device));
   };
   return CreateFunctionPass(pass_func, 1, "RewriteAnnotatedOps",
-                            {tir::StringImmNode::make("InferType")});
+                            {runtime::String("InferType")});
```

Comment:
I wonder if we can directly do {"InferType"} here now(if String's auto conversion works)

Reply:
yeah, the reason seems because Array<runtime::String/stringimm> doesn't really take initializers like {"aa", "bb"}

Reply:
If you remove the explicit keyword of String's constructor from std::string, or const char* it might work

Reply:
I tried a bit more. It looks we actually need the constructor from `const char*` in order to make the initializer_list work. Previously, we only have the constructor from `std::string`. I will add it.